components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Runtime/Extensions/SecurityConfigEx.cs

```diff
@@ -5,22 +5,25 @@
 
 namespace Microsoft.Azure.IIoT.OpcUa.Protocol {
     using Microsoft.Azure.IIoT.OpcUa.Protocol.Runtime;
-    using System;
     using Opc.Ua;
+    using Opc.Ua.Configuration;
+    using System;
+    using System.Threading.Tasks;
 
     /// <summary>
     /// Certificate store extensions
     /// </summary>
     public static class SecurityConfigEx {
 
+
         /// <summary>
-        /// Convert to security configuration
+        /// Build the security configuration
```

Comment:
It should be documented that input `applicationConfiguration` is going to be changed.

Reply:
right